lib/sidekiq/config.rb

```diff
@@ -17,10 +17,9 @@ class Config
       poll_interval_average: nil,
       average_scheduled_poll_interval: 5,
       on_complex_arguments: :raise,
-      iteration: {
-        max_job_runtime: nil,
-        retry_backoff: 0
```

Comment:
Was this unused? Should we worry about backwards compatibility?

Reply:
This was unused and you forgot to remove this part when removed this feature from the original PR, so just removing lgtm.